README.md

```diff
@@ -10,33 +12,68 @@
 [![Known Vulnerabilities](https://snyk.io/test/npm/@ice/store/badge.svg)](https://snyk.io/test/npm/@ice/store)
 [![David deps](https://img.shields.io/david/ice-lab/icestore.svg?style=flat-square)](https://david-dm.org/ice-lab/icestore)
 
-## 安装
+## Installation
 
 ```bash
 npm install @ice/store --save
 ```
 
-## 快速上手
+## Introduction
+
+icestore is a lightweight React state management library based on hooks. It has following core features:
+
+* Minimal API: only 3 apis, easy to pick up in 5 minutes
+* Predictable: unidirectional data flow just like Redux, only allow state mutation inside actions, make data flow easier to trace
+* Optimal performance: multiple store design and only trigger rerender when state changed
+* Built in async status: record loading and error status of async actions, simplify loading and error render logic in view
+
+The data flow is as follows:   
+
+<img src="https://user-images.githubusercontent.com/5419233/60878757-f44a6b00-a272-11e9-8afa-d47e8493e040.png" width="400" />
+
+### Compatibility
+
+icestore is only compatable with React 16.8.0 and later cause it's dependency of React hooks.
 
-声明 Store：
+## Getting Started
+
+Let's build a simple todo app from scatch using icestore which includes following steps:
+
+* Define store config which is a plain javascript object, properties of function type correspond to action and non function type correspond to state.   
 
 ```javascript
 // src/stores/todos.js
 export default {
   dataSource: [],
   async refresh() {
-    this.dataSource = await (await fetch(/* api */)).json();
+    this.dataSource = await new Promise(resolve =>
+      setTimeout(() => {
+        resolve([
+          {
+            name: "react"
+          },
+          {
+            name: "vue",
+            done: true
+          },
+          {
+            name: "angular"
+          }
+        ]);
+      }, 1000)
+    );  },
+  add(todo) {
+    this.dataSource.push(todo);
   },
-  async action() {
-    // ...
+  remove(index) {
+    this.dataSource.splice(index, 1);
   },
-  actionSync() {
-    // ...
+  toggle(index) {
+    this.dataSource[index].done = !this.dataSource[index].done;
   },
 };
 ```
-
-注册 Store：
+* Initialize store instance and register pre-defined store config to store instance by namespace.
```

Comment:
Initialize the store instance and register the pre-defined store config using the namespace.

Reply:
Note: Might need to explain what you mean by namespace. For example, "Refer to the API" section.